__test__/UserRegister.spec.ts

```diff
@@ -11,28 +11,30 @@ beforeEach(() => {
   return User.destroy({ truncate: true });
 });
 
-describe('User Registration', () => {
-  const postValidUser = () => {
-    return request(app).post('/api/1.0/users').send({
-      username: 'user1',
-      email: 'user1@mail.com',
-      password: 'P4ssword',
-    });
-  };
+const validUser = {
+  username: 'user1',
+  email: 'user1@mail.com',
+  password: 'P4ssword',
+};
+
+const postUser = (user: { username: string | null; email: string | null; password: string | null } = validUser) => {
```

Comment:
Added a customize postUser that can be used for mocking post request from client